process_dev.json

```diff
@@ -10,28 +10,28 @@
 		{
 			"name": "conductor",
 			"cwd": "api/src/workers",
-			"interpreter": "../../node_modules/babel-cli/bin/babel-node.js",
+			"interpreter": "../node_modules/babel-cli/bin/babel-node.js",
 			"script": "./conductor.js",
 			"watch": true
 		},
 		{
 			"name": "rss",
 			"cwd": "api/src/workers",
-			"interpreter": "../../node_modules/babel-cli/bin/babel-node.js",
+			"interpreter": "../node_modules/babel-cli/bin/babel-node.js",
 			"script": "./rss.js",
 			"watch": true
 		},
 		{
 			"name": "podcast",
 			"cwd": "api/src/workers",
-			"interpreter": "../../node_modules/babel-cli/bin/babel-node.js",
+			"interpreter": "../node_modules/babel-cli/bin/babel-node.js",
 			"script": "./podcast.js",
 			"watch": true
 		},
 		{
 			"name": "og",
 			"cwd": "api/src/workers",
-			"interpreter": "../../node_modules/babel-cli/bin/babel-node.js",
+			"interpreter": "../node_modules/babel-cli/bin/babel-node.js",
 			"script": "./og.js",
 			"watch": true
 		}
```

Comment:
This breaks process_dev.json if you follow the Winds install instructions...

At least my version of PM2 does api/src/workers../../node_modules/babel-cli/bin/babel-node.js
So the 2 `../../` is correct and `../` is wrong.

To make sure we're on the same page. RM -rf all your node modules directories. Install Node 10.5.0 and yarn install in the api dir and the app dir.